diff -Nur old/panel-plugin/Makefile.am new/panel-plugin/Makefile.am
--- old/panel-plugin/Makefile.am	2003-11-09 22:06:20.000000000 +0000
+++ new/panel-plugin/Makefile.am	2003-11-14 14:49:43.000000000 +0000
@@ -15,7 +15,7 @@
 libxmms_plugin_la_CFLAGS =						\
 	@XFCE4_PANEL_CFLAGS@						\
         `xmms-config --cflags`						\
-        -DDATA_DIR=\"/usr/local/share/xfce4/xmms-plugin\"
+        -DDATA_DIR=\"/usr/share/xfce4/xmms-plugin\"
 
 libxmms_plugin_la_LIBADD = 						\
 	@XFCE4_PANEL_LIBS@
diff -Nur old/panel-plugin/Makefile.in new/panel-plugin/Makefile.in
--- old/panel-plugin/Makefile.in	2003-11-14 10:42:39.000000000 +0000
+++ new/panel-plugin/Makefile.in	2003-11-14 14:49:50.000000000 +0000
@@ -145,7 +145,7 @@
 libxmms_plugin_la_CFLAGS = \
 	@XFCE4_PANEL_CFLAGS@						\
         `xmms-config --cflags`						\
-        -DDATA_DIR=\"/usr/local/share/xfce4/xmms-plugin\"
+        -DDATA_DIR=\"/usr/share/xfce4/xmms-plugin\"
 
 
 libxmms_plugin_la_LIBADD = \
